src/main/java/org/opentripplanner/routing/edgetype/factory/PatternHopFactory.java

```diff
@@ -497,62 +435,6 @@ private void interline(Collection<TripPattern> tripPatterns, Graph graph) {
         return null;
     }
 
-    private void loadAgencies(Graph graph) {
-        for (Agency agency : transitService.getAllAgencies()) {
-            graph.addAgency(feedId.getId(), agency);
-        }
-    }
-
-    private void loadFeedInfo(Graph graph) {
-        for (FeedInfo info : transitService.getAllFeedInfos()) {
-            graph.addFeedInfo(info);
-        }
-    }
-
-    private void loadPathways(Graph graph) {
-        for (Pathway pathway : transitService.getAllPathways()) {
-            Vertex fromVertex = stopNodes.get(pathway.getFromStop());
-            Vertex toVertex = stopNodes.get(pathway.getToStop());
-            if (pathway.isWheelchairTraversalTimeSet()) {
-                new PathwayEdge(fromVertex, toVertex, pathway.getTraversalTime(), pathway.getWheelchairTraversalTime());
-            } else {
-                new PathwayEdge(fromVertex, toVertex, pathway.getTraversalTime());
-            }
-        }
-    }
-
-    private void loadStops(Graph graph) {
-        for (Stop stop : transitService.getAllStops()) {
-            // Add a vertex representing the stop.
-            // FIXME it is now possible for these vertices to not be connected to any edges.
-            TransitStopVertex stopVertex = new TransitStopVertex(graph, stop);
-            stopNodes.put(stop, stopVertex);
-        }
-        for (Station station : transitService.getAllStations()) {
-            // TODO: What to do about linking transit vertices directly to stations
-            /*
-            stopNodes.put(station, new TransitStation(graph, station));
-             */
-        }
-    }
-
-    private void clearCachedData() {
-        LOG.debug("shapes=" + geometriesByShapeId.size());
-        LOG.debug("segments=" + geometriesByShapeSegmentKey.size());
-        geometriesByShapeId.clear();
-        distancesByShapeId.clear();
-        geometriesByShapeSegmentKey.clear();
-    }
-
-    private void loadTransfers(Graph graph) {
-        Collection<Transfer> transfers = transitService.getAllTransfers();
-        TransferTable transferTable = graph.getTransferTable();
-        for (Transfer sourceTransfer : transfers) {
-            transferTable.addTransfer(sourceTransfer);
-        }
-    }
```

Comment:
The PatternHopFactory does not create hop edges anymore, and with this PR it no longer loads any transit data into the Graph. It just post-processes the data already in the graph to prepare geometries for the TripPatterns. The Javadoc, name, and log entries of the class should be updated accordingly. I would suggest something like ProcessPatternGeometriesAndBlocks, with a similar setup to AddTransitModelEntitiesToGraph (concrete class with a static method that instantiates and triggers the action).